extract_display_quotes.py

```diff
@@ -158,7 +181,11 @@ def load_txt(self, value: dict) -> list:
         temp = {'text_name': value['metadata']['name'][:-4],
                 'text': codecs.decode(value['content'], encoding='utf-8', errors='replace')
         }
-    
+        
+        unknown_count = temp['text'].count('�')
```

Comment:
What is "�"? Will all non-unicode character be counted with this symbol? This hardcoding doesn't look very reliable.